index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <h1 id="title">Survey Form</h1>
+    <p id="description">Form project for Microverse</p>
+    <form action="" id="survey-form">
+      <div>
+        <label for="name" id="name-label">Name:</label>
+        <input type="text" id="name" placeholder="Your Name" required />
+      </div>
+      <div>
+        <label for="email" id="email-label">Email:</label>
+        <input type="email" id="email" placeholder="Your email" required />
+      </div>
+      <div>
+        <label for="number" id="number-label">Number:</label>
+        <input
+          type="number"
+          id="number"
+          min="10"
+          max="1000"
+          placeholder="Enter a number"
+        />
+      </div>
+      <div>
+        <label for="dropdown">Job:</label>
+        <select name="job" id="dropdown">
+          <option value="student">Student</option>
+          <option value="engineer">Engineer</option>
+          <option value="developer">Developer</option>
+        </select>
+      </div>
+      <div>
+        <label for="male">Male</label>
+        <input type="radio" name="gender" id="male" value="Male" />
+        <label for="female">Female</label>
+        <input type="radio" name="gender" id="female" value="Female" />
```

Comment:
- [ ] This review is great as well, kindly state the reason(s) for the required change; this will definitely improve the developer programming skills. 👍